.github/workflows/devnet-deploys.yml

```
@@ -192,6 +212,12 @@ jobs:
               echo "bot_follow_chain=PROVEN" >> $GITHUB_OUTPUT
               echo "bot_flush_setup_txs=false" >> $GITHUB_OUTPUT
               echo "bot_max_pending_txs=1" >> $GITHUB_OUTPUT
+              echo "mainnet_fork_cpu_units=2048" >> $GITHUB_OUTPUT
+              echo "mainnet_fork_memory_units=4096" >> $GITHUB_OUTPUT
+              echo "bot_skip_simulation=true" >> $GITHUB_OUTPUT
+              echo "bot_l2_gas_limit=1000000000" >> $GITHUB_OUTPUT
+              echo "bot_da_gas_limit=1000000000" >> $GITHUB_OUTPUT
+              echo "bot_count=4" >> $GITHUB_OUTPUT
```

```suggestion
              echo "bot_skip_simulation=false" >> $GITHUB_OUTPUT
              echo "bot_l2_gas_limit=" >> $GITHUB_OUTPUT
              echo "bot_da_gas_limit=" >> $GITHUB_OUTPUT
              echo "bot_count=1" >> $GITHUB_OUTPUT
```
I'd say let's roll back alphanet to match devnet config